docs/common/yum.md

```diff
@@ -1,46 +1,87 @@
-YUM Repositories
+OSG Yum Repositories
 ====================
 
-About This Document
--------------------
-
-This document introduces YUM repositories and how OSG uses them.
-If you are unfamiliar with YUM, see the [documentation on using YUM and RPM](/release/yum-basics).
+This document introduces Yum repositories and how they are used in the OSG.
+If you are unfamiliar with Yum, see the [documentation on using Yum and RPM](/release/yum-basics).
 
 Repositories
 ------------
 
-OSG hosts four public-facing repositories at [repo.opensciencegrid.org](https://repo.opensciencegrid.org/):
+The OSG hosts multiple repositories at [repo.opensciencegrid.org](https://repo.opensciencegrid.org/osg/) that are
+intended for public use:
 
--   **release**: RPMs considered production-ready.
--   **testing**: RPMs not yet ready for release; expect bugs.
--   **development**: RPMs that are bleeding-edge;
-      do not use without instruction from OSG Software and Release team members.
--   **contrib**: RPMs contributed from outside the OSG S&R team;
-      no official OSG support.
--   **upcoming release/testing/development**: Similar to release/testing/development but for new versions of software that may require manual action after an update.
+| Yum repository                                                                             | Contains RPMs that...                                                                                                                |
+|--------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| `osg`                                                                                      | Are considered production-ready (default).                                                                                           |
```

Comment:
I think it's a little inconsistent that the header is not a complete sentence but the content starts with a capital letter -- "Contains RPMs that Are considered production-ready"